int8 left_motor
int8 right_motor
int8 vertical_motor
